tools/clang/test/Dxil2Spv/passthru-cs.ll

```diff
@@ -82,94 +82,93 @@ attributes #2 = { nounwind }
-; %type_ByteAddressBuffer = OpTypeStruct %_runtimearr_uint
-; %_ptr_Uniform_type_ByteAddressBuffer = OpTypePointer Uniform %type_ByteAddressBuffer
-; %type_RWByteAddressBuffer = OpTypeStruct %_runtimearr_uint
-; %_ptr_Uniform_type_RWByteAddressBuffer = OpTypePointer Uniform %type_RWByteAddressBuffer
-;      %v3uint = OpTypeVector %uint 3
-; %_ptr_Input_v3uint = OpTypePointer Input %v3uint
-;        %void = OpTypeVoid
-;          %19 = OpTypeFunction %void
-;         %int = OpTypeInt 32 1
-; %dx_types_ResRet_i32 = OpTypeStruct %int %int %int %int %int
-; %_ptr_Function_dx_types_ResRet_i32 = OpTypePointer Function %dx_types_ResRet_i32
-; %_ptr_Input_uint = OpTypePointer Input %uint
-; %_ptr_Uniform_uint = OpTypePointer Uniform %uint
-; %_ptr_Function_int = OpTypePointer Function %int
-;           %3 = OpVariable %_ptr_Uniform_type_ByteAddressBuffer Uniform
-;           %4 = OpVariable %_ptr_Uniform_type_RWByteAddressBuffer Uniform
-; %gl_GlobalInvocationID = OpVariable %_ptr_Input_v3uint Input
-;        %main = OpFunction %void None %19
-;          %20 = OpLabel
-;          %24 = OpVariable %_ptr_Function_dx_types_ResRet_i32 Function
-;          %26 = OpAccessChain %_ptr_Input_uint %gl_GlobalInvocationID %uint_0
-;          %27 = OpLoad %uint %26
-;          %28 = OpShiftLeftLogical %uint %27 %uint_2
-;          %29 = OpIAdd %uint %28 %uint_0
-;          %31 = OpAccessChain %_ptr_Uniform_uint %3 %uint_0 %29
-;          %32 = OpLoad %uint %31
-;          %34 = OpAccessChain %_ptr_Function_int %24 %uint_0
-;          %35 = OpBitcast %int %32
-;                OpStore %34 %35
-;          %36 = OpIAdd %uint %28 %uint_1
-;          %37 = OpAccessChain %_ptr_Uniform_uint %3 %uint_0 %36
-;          %38 = OpLoad %uint %37
-;          %39 = OpAccessChain %_ptr_Function_int %24 %uint_1
-;          %40 = OpBitcast %int %38
-;                OpStore %39 %40
-;          %41 = OpIAdd %uint %28 %uint_2
-;          %42 = OpAccessChain %_ptr_Uniform_uint %3 %uint_0 %41
-;          %43 = OpLoad %uint %42
-;          %44 = OpAccessChain %_ptr_Function_int %24 %uint_2
-;          %45 = OpBitcast %int %43
-;                OpStore %44 %45
-;          %46 = OpIAdd %uint %28 %uint_3
-;          %47 = OpAccessChain %_ptr_Uniform_uint %3 %uint_0 %46
-;          %48 = OpLoad %uint %47
-;          %49 = OpAccessChain %_ptr_Function_int %24 %uint_3
-;          %50 = OpBitcast %int %48
-;                OpStore %49 %50
-;          %51 = OpIAdd %uint %28 %uint_4
-;          %52 = OpAccessChain %_ptr_Uniform_uint %3 %uint_0 %51
-;          %53 = OpLoad %uint %52
-;          %54 = OpAccessChain %_ptr_Function_int %24 %uint_4
-;          %55 = OpBitcast %int %53
-;                OpStore %54 %55
-;          %56 = OpAccessChain %_ptr_Function_int %24 %uint_0
-;          %57 = OpAccessChain %_ptr_Uniform_uint %4 %uint_0 %28
-;          %58 = OpBitcast %uint %56
-;                OpStore %57 %58
-;                OpReturn
-;                OpFunctionEnd
+; CHECK:      ; SPIR-V
```

Comment:
Do you plan to clean up tests in future PRs? I would not expect them to try to match every single output line

Reply:
I hadn't. The plan was that we have "WholeFileTests" for just the 3 passthrough shaders, and their small benefit was that they do provide a reference for the entire output (which was probably more helpful for the initial development up until now). We have the same on the SPIR-V backend and I haven't yet seen a change that required modifications to the handful of them, so I don't expect the churn to be too bad but if it is, then changing them makes sense. If they seem like more harm than good though let me know and I'll track a follow up.

Reply:
I think this is good enough for what we need now, but some things seem too specific. For example, I don't think we care about the exact ID numbers that are printed, e.g., `%19 = OpTypeFunction %void`. If this becomes a problem in the future and we need to update these files often, we can revisit this at that time.